lib/webdriveragent.js

```diff
@@ -286,6 +300,20 @@ class WebDriverAgent {
       return await this.getStatus();
     }
 
+    if (this.isRealDevice && this.usePreinstalledWDA) {
+      const xctestEnv = {
+        USE_PORT: this.wdaLocalPort || WDA_AGENT_PORT
```

Comment:
do we only care about this single param?

Reply:
Added WDA_PRODUCT_BUNDLE_IDENTIFIER and MJPEG_SERVER_PORT as well. Maybe they are also reasonable